apps/new/widget/page/home/Hero.jsx

```diff
@@ -777,7 +777,10 @@ const Hero = () => {
           Designed to support builders in a multi-chain ecosystem.
         </Subheading>
         <div className="d-flex align-items-center gap-4 hero-btns">
-          <Button className="hero-button" href={"?page=projects&tab=editor"}>
+          <Button
+            className="hero-button"
+            href={`${context.account ? "${config_index}?page=projects" : "${alias_gateway_url}/join"}`}
+          >
```

Comment:
There were landing page tests that had been missing, now added back, showing that this change shouldn't be here

Please revert this